modules/@apostrophecms/notification/index.js

```diff
@@ -250,14 +249,7 @@ module.exports = {
                 dismissed: []
               };
             }
-            const result = await self.find(req, {});
-            const notifications = result.notifications;
-            const dismissed = result.dismissed;
-
-            if (!notifications.length && !dismissed.length) {
```

Comment:
You broke the long polling here. It's supposed to hang out and keep retrying until it hits the longPollingTimeout or finds an updated/modified notification to send.

Reply:
That way it is more responsive than the 5 second interval would suggest.

Reply:
Done